src/util/MongoBinary.js

```diff
@@ -22,12 +22,23 @@ export default class MongoBinary {
   static cache: MongoBinaryCache = {};
 
   static async getPath(opts?: MongoBinaryOpts = {}): Promise<string> {
-    const {
-      downloadDir = path.resolve(os.homedir(), '.mongodb-binaries'),
-      platform = os.platform(),
-      arch = os.arch(),
-      version = 'latest',
-    } = opts;
+    const defaultOptions = {
+      downloadDir: process.env.hasOwnProperty('MONGOMS_DOWNLOAD_DIR')
+        ? process.env.MONGOMS_DOWNLOAD_DIR
+        : path.resolve(os.homedir(), '.mongodb-binaries'),
+      platform: process.env.hasOwnProperty('MONGOMS_PLATFORM')
+        ? process.env.MONGOMS_PLATFORM
+        : os.platform(),
+      arch: process.env.hasOwnProperty('MONGOMS_ARCH') ? process.env.MONGOMS_ARCH : os.arch(),
+      version: process.env.hasOwnProperty('MONGOMS_VERSION')
+        ? process.env.MONGOMS_VERSION
+        : 'latest',
+      debug: process.env.hasOwnProperty('MONGOMS_DEBUG')
```

Comment:
For error fix you need to change this line:
```diff
- process.env.hasOwnProperty('MONGOMS_DEBUG')
+ (typeof process.env.MONGOMS_DEBUG === 'string')
```

Reply:
When I merge changes to master Travis will publish a new package. With broken tests - will not. So Travis tests should pass without errors in your PR.

You may run `yarn test` for locally testing on your dev machine.

Reply:
It's very strange fix! =)
I wonder how it affected to tests?

Reply:
I will not resolve it fix because it is not valid for eslint. I make explicit commit.

Reply:
Hm, I'll clone your fork and look at problem closely.